_data/committers.yml

```diff
@@ -139,14 +159,15 @@
   role: PMC
   alias: tdunning
   affiliation: MapR
+- name: Todd Lipcon
```

Comment:
FYI I think I'm considered "emeritus" on Arrow at the moment.

Reply:
Hmm, I don't know how to query emeritus members from the Apache directory. Do we have a well-known list of emeritus Arrow members?

Reply:
One can query the repositories for activity and if no activity for a certain time period, indicate someone as inactive.  This would require updating committer list to have Jira and GitHub ids, though could be done manually at present until a policy is formulated.

Reply:
I don't think we want to do that. Going "emeritus" requires an official Apache procedure: https://www.apache.org/foundation/bylaws.html#4.2